JASP-Engine/JASP/R/jagsModule.R

```diff
@@ -720,7 +674,7 @@ JAGS <- function(jaspResults, dataset, options, state = NULL) {
   )
 
   if (length(toAdd) > 0L)
-    errorMessage <- paste0(errorMessage, "\n\nIn addition:\n", toAdd)
+    errorMessage <- paste0(errorMessage, "\n\n", gettext("In addition:"), "\n", toAdd)
```

Comment:
This whole thing should probably be a `gettextf`  for those languages that do not go left->right

Reply:
This is not so much left to right, but rather top to bottom. I'll still change it though. Shall I escape the newlines?

Reply:
Yeah you can put those outside the gettext